styles/prosilver/template/index_body.html

```diff
@@ -2,7 +2,46 @@
 
 <h2>{{ lang('CUSTOMISATION_DATABASE') }}</h2>
 
-	<div class="action-bar contrib-filter" style="margin-top: 2em;">
+	{% if U_IS_TITANIA_INDEX %}
+	<div class="contrib-featured-container">
+		{% for contribs in loops.featured_contribs %}
+		<div class="contrib-quickview contrib-quickview-featured">
+			<div class="contrib-featured">
+				<span>{{ lang('FEATURED_CONTRIBUTION') }}</span>
+			</div>
+			<a href="{{ contribs.U_VIEW_CONTRIB }}" class="quickview-title">{{ contribs.CONTRIB_NAME }}</a>
+			<div class="quickview-preview">
+				<div class="quickview-desc">{{ contribs.DESC_SNIPPET }}</div>
+				<div class="quickview-image">
+					{% if contribs.PREVIEW %}
+					<img src="{{ contribs.PREVIEW }}" alt="{{ contribs.CONTRIB_NAME }}" class="screenshot" />
+					{% else %}
+					<img src="{{ T_TITANIA_THEME_PATH }}/images/types/{{ contribs.CONTRIB_TYPE_CLEAN }}.png" alt="{{ lang('NO_SCREENSHOT') }}" class="screenshot" />
+					{% endif %}
+				</div>
+			</div>
+			{{ contribs.CONTRIB_RATING_STRING }}
+			<br />
+			{{ lang('BY') }} {{ contribs.AUTHOR_NAME_FULL }}<br /><br />
```

Comment:
Would rather not use `br` tags. Can you try using `p` tags around the above instead and around the for loop that follows?